### DISCUSSION TITLE
ssh connection to secondary server

### DISCUSSION THREAD

**Discussion by cirobr** [microsoft/vscode-discussions#415] [Extension Development QnA] [answered]
Cheers,

In order to connect to a secondary server through a primary server, from my own PC, I issue the following command on a Linux terminal:

`ssh -t user@server1.com ssh user@server2.com`

How do I use such command chain on VSCode? How do I store it in config file?

Thanks in advance.

**Comment by isidorn** [comment, marked as answer]
I suggest to ask this on StackOverflow. This discussion is primarily for extension authors.
